src/main/java/games/poker/PokerForwardModel.java

```diff
@@ -164,6 +170,8 @@ protected void _afterAction(AbstractGameState gameState, AbstractAction action)
                     // Round is over
                     roundEnd(pgs);
                 }
+
+                pgs.resetTurnOwner();
```

Comment:
You don't actually need to set up a new method for this - you should use the framework method (on StandardForwardModel) of endPlayerTurn(gameState, int newTurnOwner).
Or, if we just move on to the next clockwise player (who has not been knocked out), then endPlayerTurn(gameState) will do this as default behaviour.